src/lib/homestore_backend/hs_blob_manager.cpp

```diff
@@ -104,6 +104,11 @@ BlobManager::AsyncResult< blob_id_t > HSHomeObject::_put_blob(ShardInfo const& s
         return folly::makeUnexpected(BlobError(BlobErrorCode::NOT_LEADER, repl_dev->get_leader_id()));
     }
 
+    if (!repl_dev->is_ready_for_traffic()) {
+        LOGW("failed to put blob for pg [{}], shard [{}], not ready for traffic", pg_id, shard.id);
+        return folly::makeUnexpected(BlobError(BlobErrorCode::UNSUPPORTED_OP));
```

Comment:
better if it can be `TRY_AGAIN`

Reply:
`TRY_AGAIN` seems to be used in somewhere (has compile confilct), so I update to `RETRY_REQUEST`